app/package.json

```diff
@@ -8,9 +8,10 @@
     "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "lint": "next lint",
+    "postinstall": "cd .. && husky install app/.husky",
```

Comment:
Forgive my ignorance, should this be post-install to follow precedent set with format-check and ts-check, or is postinstall a known singular command? Does it matter?

Reply:
`post[x]` has a special meaning in `package.json` where it runs after `[x]` runs, so after `install` runs, `postinstall` will run. if it's named `post-install`, then it would run automatically after `-install` runs. so it has to be `postinstall` in this case.

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts

Reply:
that is npm docs but i believe yarn's interface is compatible

Reply:
Thank you, I had a feeling there was something more to it.